### PR TITLE
[Modules] Adding Standard SKU Options to bastionHosts

### PR DESCRIPTION
**Pull request by ChrisSidebotham**
# Description

Addition of standard SKU options to support tunnelling, native client and enable/disable features.

| Pipeline |
| - |
| [Network: BastionHosts](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.bastionhosts.yml) |

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

File: modules/Microsoft.Network/bastionHosts/.test/parameters.json [hunk 1]
```diff
@@ -20,6 +20,18 @@
...
+        },
+        "enableFileCopy": {
+            "value": false
+        },
+        "enableIpConnect": {
+            "value": false
+        },
+        "enableShareableLink": {
+            "value": false
+        },
```
Comment (ahmadabdalla) [inline modules/Microsoft.Network/bastionHosts/.test/parameters.json:34]: @ChrisSidebotham can we add a min.parameters.json to validate the minimum required parameters for a bastion host? Probably have it as basic sku?

**Review by ahmadabdalla — CHANGES_REQUESTED** [review]
Just one comment regarding test coverage for the minimum set of parameters

File: modules/Microsoft.Network/bastionHosts/.test/parameters.json (hunk 1 above)
Comment (ChrisSidebotham) [reply modules/Microsoft.Network/bastionHosts/.test/parameters.json:34]: We already have a min.parameters.json test in the .tests folder https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.Network/bastionHosts/.test/min.parameters.json 
Is this what you are referring to?

Comment (ahmadabdalla) [reply modules/Microsoft.Network/bastionHosts/.test/parameters.json:34]: Totally missed this one. Sorry for that

**Review by ahmadabdalla — APPROVED** [review]

**Pull request merged**